paystell-frontend/src/components/PaymentLinkScreen/page.tsx

```diff
@@ -0,0 +1,19 @@
+import { Header } from "@/components/PaymentLinkScreen/Components/header";
+import { ProductInfo } from "@/components/PaymentLinkScreen/Components/ProductInfo";
+import { PaymentSummary } from "@/components/PaymentLinkScreen/Components/PaymentSummary";
+import { PaymentButton } from "@/components/PaymentLinkScreen/Components/PaymentButton";
+import { Footer } from "@/components/PaymentLinkScreen/Components/footer";
```

Comment:
Components should remain as requested within src/components/PaymentLinkScreen without the Components folder
```suggestion
import { Header } from "@/components/PaymentLinkScreen/header";
import { ProductInfo } from "@/components/PaymentLinkScreen/ProductInfo";
import { PaymentSummary } from "@/components/PaymentLinkScreen/PaymentSummary";
import { PaymentButton } from "@/components/PaymentLinkScreen/PaymentButton";
import { Footer } from "@/components/PaymentLinkScreen/footer";
```

Reply:
I have just done that. Kindly check the push made